acceptance/srv-configs/clusterman-clusters/local-dev/default.kubernetes

```diff
@@ -15,3 +15,7 @@ scaling_limits:
 autoscale_signal:
   internal: true
   period_minutes: 1
+
+autoscaling:
+  prevent_scale_down_after_capacity_loss: true
+  capacity_loss_threshold: 3
```

Comment:
since this is _not_ weight-based (and almost everything else is) can we rename this to make it clear that it's based on the number of EC2 instances lost?

Reply:
Will rename to `instance_loss_threshold`